src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -302,7 +295,7 @@ export function getStatusContents(statusType, details = {}, name = {}) {
       contents.title = 'The judge is seeking additional information before making a decision';
       contents.description = (
         <p>The Board of Veterans’ Appeals is seeking evidence or an outside opinion from a legal,
-        medical, or other professional necessary to make decision about your appeal.</p>
```

Comment:
- Change "necessary to" to "in order to":
"medical, or other professional in order to make a decision about your appeal."